test/lib/Chainweb/Test/TestVersions.hs

```diff
@@ -158,7 +158,7 @@ testVersionTemplate v = v
     & versionHeaderBaseSizeBytes .~ 318 - 110
     & versionWindow .~ WindowWidth 120
     & versionMaxBlockGasLimit .~ Bottom (minBound, Just 2_000_000)
-    & versionMinimumBlockHeaderHistory .~ Bottom (minBound, Just 100)
+    & versionMinimumBlockHeaderHistory .~ Bottom (minBound, Just 20)
```

Comment:
this smaller number is more manageable for tests, individual tests could change it if they want to.